ArduCopter/motors.pde

```diff
@@ -191,9 +173,6 @@ static bool init_arm_motors(bool arming_from_gcs)
     // enable output to motors
     output_min();
 
-    // finally actually arm the motors
-    motors.armed(true);
-
     // log arming to dataflash
     Log_Write_Event(DATA_ARMED);
```

Comment:
Probably not. Depends of our definition of arming anyway (which I took as "motors start spinning")